### PR TITLE
Removed LV_EVENT_PRESSED/RELEASED related event in lv_checkbox

### PR DESCRIPTION
**Pull request by mariotaku**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [x] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

**Comment by kisvegabor** [comment]
Does this invalidation is required at all? It seems working well without it too. IIRC it's a leftover from a previous mechansim. 

I've just updated the PR template because CHANGELOG shouldn't be updated manually anymore as we generate it automatically.

**Comment by mariotaku** [comment]
Not sure about the invalidation part, maybe it can be included in another patch?

**Comment by mariotaku** [comment]
Just saw related logic also exists in `lv_obj` and I have removed this duplicated part.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ Some of the most important features are
 
 ### Fixes
 
+- fix(checkbox): fixed LV_EVENT_VALUE_CHANGED event [`2829`](https://github.com/lvgl/lvgl/pull/2829)
```
Comment (kisvegabor) [inline docs/CHANGELOG.md]:
```suggestion
```
It should be removed.

**Comment by kisvegabor** [comment]
Looks good, thanks!

**Pull request merged**